sshconnect.c

```diff
@@ -391,6 +391,9 @@ ssh_create_socket(struct addrinfo *ai)
 		}
 		memcpy(&bindaddr, res->ai_addr, res->ai_addrlen);
 		bindaddrlen = res->ai_addrlen;
+#ifdef IP_BIND_ADDRESS_NO_PORT
```

Comment:
Isn't this call unnecessary? Couldn't you have just the single `setsockopt()` call before the final `bind()` call?